src/App.js

```diff
@@ -1,25 +1,40 @@
-import logo from './logo.svg';
 import './App.css';
+import TopBar from './components/TopBar';
+import SideBar from './components/SideBar';
+import Login from './components/login';
+import Register from './components/Register';
+import { useState } from 'react';
 
-function App() {
+const App = () => {
+  const [showSideBar, setShowSideBar] = useState(false);
+  const [visibleLogin, setVisibilityLogin] = useState(true);
+  const [visibleRegister, setVisibilityRegister] = useState(false);
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <TopBar
+        setShowSideBar={setShowSideBar}
+        visibleLogin={visibleLogin}
+        setVisibilityLogin={setVisibilityLogin}
+        visibleRegister={visibleRegister}
+        setVisibilityRegister={setVisibilityRegister}
```

Comment:
Also, passing these as props is ok for this task, but in the next we'll want to refactor it so they are stored in a context 👍